Cargo.toml

```diff
@@ -4,38 +4,28 @@ version = "0.1.0"
 authors = ["Victor Lopez <victor@dusk.network>"]
 edition = "2018"
 
+[lib]
+name = "rusk"
+path = "src/rusk/lib.rs"
```

Comment:
Maybe "src/lib/lib.rs" instead of "src/rusk/lib.rs"? I don't know, it's just to have consistency with "src/bin/" but I don't mind either way

Reply:
Makes sense.